<launch>   
    <node name="forward_octomag" pkg="mag_calculator" type="forward_model">
        <param name="system_name" type="string" value="OctoMag" />
        <param name="calibration_path" type="string" value="$(find mpem)/cal/OctoMag_Calibration.yaml" />
        <param name="calibration_type" type="string" value="mpem" />
    </node>

    <node pkg="tf" type="static_transform_publisher" name="mns_broadcaster" args="0 0 0 0 0 0 1 world mns 100" />
</launch>
